controllers/status.go

```diff
@@ -137,9 +137,11 @@ func (r *SpecialResourceReconciler) clusterOperatorStatusUpdate() error {
 	return nil
 }
 
+//FIXME: grep for all other locations in which openshift-special-resource-operator is hardcoded
 func (r *SpecialResourceReconciler) clusterOperatorUpdateRelatedObjects() error {
 	relatedObjects := []configv1.ObjectReference{
-		{Group: "", Resource: "namespaces", Name: "openshift-special-resource-operator"},
+		//FIXME: put in SpecialResourceReconciler object
+		{Group: "", Resource: "namespaces", Name: os.Getenv("NAMESPACE")},
```

Comment:
I would rather have this as `r.Namespace` and initialized in main.

Reply:
I agree, this is what the `FIXME` is for.

There are a lot of places in which we just access env variables, I have create a [task](https://issues.redhat.com/browse/MGMT-8672) to fix it vertically on all our code and all our env variables.